### ISSUE TITLE
[a11Y Support] Accessibility Community of Practice with CAIA: Transfer backlog items from a11y dashboard into CAIA tickets

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#56174]
> [!CAUTION] 
> This is an older ticket; however, it is a CAIA `Chipper` ticket, that can be completed over time, and it will need to be revisited with VA accessibility specialists in the Accessibility Community of Practice - including @artsymartha68 to determine next steps.  
> 
> **In an effort to update this ticket - a task list was added on 7/18/2024. Initial next steps could be:** 
> - The list of planned backlog items from Josh's spreadsheet was brought over into this ticket.
> - We likely will need to review and include for any accomplishments that have been completed, planned or are in progress since Josh's departure. 
> - Add to the Accessibility Community of Practice meeting agenda. 
> - Discuss in a future A11Y CAIA/OCTO meeting. 

## Asignee(s)
- CAIA
- Accessibility Community of Practice
- Martha

## Original Ticket Description

> Currently, [April 2023] large (proactive) a11y projects have [remained static in our a11y dashboard under the backlog tab](https://docs.google.com/spreadsheets/d/1lgzMiZZ-0rBPDuB2gRupBAjsyF0WTe005VvEHYrMxww/edit?usp=sharing). We hypothesize that by moving those larger projects into epics onto the CAIA backlog, we can better:
> - publicly display and track the amount of uncompleted projects/work the CAIA team has not yet taken on 
> - better prioritize and assign projects to specialists
> 
> **This is done with the heavy caveat that:**
> - We know github projects and zenhub is not usable (and at times inaccessible) to screen reader users
> 

### GitHub Progress
**GitHub has undergone accessibility updates since the origination of this ticket.**
- Example: A GitHub Article [shared by Martha in OCTO/DSVA Slack 7/10/2024](https://dsva.slack.com/archives/C0552U2L30S/p1720613651438409) 
    - [Detailed article about how GitHub created an accessible drag-and-drop feature](https://github.blog/2024-07-09-exploring-the-challenges-in-creating-an-accessible-sortable-list-drag-and-drop/)
- [Additional/Recent GitHub Accessibility-Related Efforts](https://github.blog/tag/accessibility/)
- CAIA also did a run through of GitHub projects with a screen reader in early July 2024.

### Completed
As of July 2024

Backlog | Priority | Status | Who can work on this / is working on this
-- | -- | -- | --
Create an a11y statement and process to collect a11y feedback from the public | Done | Complete | Josh (no longer with VA)
Create VA a11y practice dashboard | Done | Complete | Josh (no longer with VA)
Revise meeting cadence and agenda for a11y specialists | Done | Complete | Josh (no longer with VA)
A11y product health dashboard  | Done | Complete, **unused** | Josh (no longer with VA)
Create and publish a11y strategy document to platform website | Done | Complete | A11Y Community of Practice
Create new platform a11y orientation | Done | Complete | Brian
A11y champions program at the VA | Done | Complete | Josh (no longer with VA)
Standardize a11y annotation resources for designers | Done | Complete | A11Y Community of Practice
Create documentation for how we do a11y reviews to calibrate with each other | Done | Complete | CAIA
Request access to mobile test devices | Done | Complete | CAIA
Create new a11y specialist orientation for a11y specialists and publish to platform website | Done | Complete | Sara
Create an a11y strategy for the VA design system | Done | Complete | Jasmine, Josh (no longer with VA), DK, Brian, Laura,
Update platform standards to WCAG 2.2+ or WCAG 2.1 AAA | Done | Complete | Brian, DK, Eli (no longer with VA) 
Publish a11y guide for product teams to Platform | Done | Complete | Jamie, Sara, Martha
Publish guide for new a11y specialists to GitHub | Done | Complete | Jamie, Sara 
Create a slackbot based on github labels to announce design intent and midpoint reviews.a | Done | Complete | Brian 

### High Priority 
As of July 2024

Backlog | Priority | Status | Who can work on this / is working on this
-- | -- | -- | --
Conduct research and improve where we store resources, documentation, and more. | High | In progress | Josh (no longer with VA), Sara S., Brian, Martha
Create a VA.gov a11y maturity model for teams | High | In progress | Eli (no longer with VA)
Create a participatory co-design program for deeper work with disabled Veterans and project teams | High | In progress | A11Y Community of Practice and Martha

### Medium Priority 
As of July 2024

Backlog | Priority | Status | Who can work on this / is working on this
-- | -- | -- | --
Investigate how disabled Veterans report accessibility issues | Medium | In progress | Josh (no longer with VA)
Publish screen reader + research troubleshooting instructions | Medium | In progress | Sara, SK
Create guidance, methods, and tooling for accessible wireframes and UAT | Medium | In progress | Jasmine, Jeana (**who is working on UAT?**)

### No Priority 
As of July 2024
Backlog | Priority | Status | Who can work on this / is working on this
-- | -- | -- | --
Operationalize a list of generative questions for research with disabled Veterans | No priority listed | No status listed | (TRANSITIONING TO INCLUSIVE RESEARCH)
Audit and create a list of inaccessible internal tools, websites, and resources | No priority listed | No status listed |  
Publish how we do what we do so others can learn from us | No priority listed | **anything else to do here?** |  
Add an "accessibility" option to the platform support bot | No priority listed | No status listed | (Note from Brian: This is politically complicated) 
VA.gov performance dashboard | No priority listed | No status listed |  
Accessibility focused training for content writers using Drupal | No priority listed | No status listed |  
Analyze 555 open a11y tickets (DOMO) | No priority listed | No status listed |  
A11y month | No priority listed | No status listed |  
Conduct generative, service-level qualitative research on the blinded Veteran experience | No priority listed | No status listed |  
Update a11y specialist ticketing templates and process documentation. | No priority listed | No status listed |  

```[tasklist]
### Definition of done
- [x] The list of planned backlog items from Josh's spreadsheet was brought over into this ticket on 7/18/2024 by @sara-amanda
- [x] Ticket Renamed
- [ ] Add to a future A11Y agenda to discuss plan of action
- [ ] Schedule meeting with Martha and CAIA A11Ys to discuss plan of action
- [ ] Determine which backlog projects should be kept or archived.
- [x] Update with additional completed accomplishments from A11Ys since August 2023.
- [x] Update with additional in progress accomplishments from A11Ys since August 2023.
- [ ] Update with additional planned work by A11Ys since August 2023.
- [ ] For the proposed active a11y work dashboard: Do we need a list of active members with start and end dates to reflect their contributions, so they can be recognized?
- [ ] Transfer all kept backlog projects into CAIA tickets with the labesl `Sitewide CAIA` `Sitewide Accessibility`
- [ ] Do we need a specific Accessibility Community of Practice label to add to these cross-collaborative efforts? Reports can be pulled based on labels.
- [ ] Connect new tickets created via the task list below, by entering a hashtag and the ticket number.
```

```[tasklist]
### New Tickets Created from Backlog
- [ ] Connect new tickets created via this task list, by entering a hashtag and the ticket number.
```

**Comment by sara-amanda** [comment]
## Ticket Resurrected & Revitalized (2026)

### Accessibility Community of Practice
Discussed in the **Accessibility Community of Practice** meeting today, January 29, 2026.

ADE's @coforma-jamie and @sara-amanda [created a spreadsheet](https://docs.google.com/spreadsheets/d/1OvI2ikjk76WD4_-N5Z7HLlcCyODJdgE2hC2R4LHywWE/edit?gid=653532263#gid=653532263) with the items listed in this ticket from July 2024, to help advance the process of updating, closing, prioritizing, adding ticket links, adding additional context, and determining action items for the remaining items. 

#### Feedback Solicited
The **Accessibility Community of Practice** was invited to participate in this exercise and leave comments in the spreadsheet, if they would like to participate in any of these initiatives, or if they would like to share any feedback and/or historical information.  

## Large Accessibility Projects 
> [!TIP]
> - **Source of Truth:** Refer to the [spreadsheet](https://docs.google.com/spreadsheets/d/1OvI2ikjk76WD4_-N5Z7HLlcCyODJdgE2hC2R4LHywWE/edit?gid=653532263#gid=653532263) as the source of truth. 
> - **Additional Information:** Detailed notes, comments and current status flags can be found in the spreadsheet. 
> - **Tickets**: Tickets should be made and linked in the spreadsheet when work from these initiatives are initiated. 

### Completed 
_Cumulative list of completed items. Includes items from July 2024 update and January 2026 update._

Backlog | Priority | Status | Who can work on this / is working on this | Status as of
-- | -- | -- | -- | --
Create an a11y statement and process to collect a11y feedback from the public | Done | Complete | Josh (no longer with VA) | July 2024 Ticket
Create VA a11y practice dashboard | Done | Complete | Josh (no longer with VA) | July 2024 Ticket
Revise meeting cadence and agenda for a11y specialists | Done | Complete | Josh (no longer with VA) | July 2024 Ticket
A11y product health dashboard | Done | Complete, unused | Josh (no longer with VA) | July 2024 Ticket
Create and publish a11y strategy document to platform website | Done | Complete | A11Y Community of Practice | July 2024 Ticket
Create new platform a11y orientation | Done | Complete | Beth | July 2024 Ticket
A11y champions program at the VA | Done | Complete | Josh (no longer with VA) | July 2024 Ticket
Standardize a11y annotation resources for designers | Done | Complete | A11Y Community of Practice | July 2024 Ticket
Create documentation for how we do a11y reviews to calibrate with each other | Done | Complete | CAIA | July 2024 Ticket
Request access to mobile test devices | Done | Complete | CAIA | July 2024 Ticket
Create new a11y specialist orientation for a11y specialists and publish to platform website | Done | Complete | Sara | July 2024 Ticket
Create an a11y strategy for the VA design system | Done | Complete | Jasmine, Josh (no longer with VA), DK, Beth, Laura, | July 2024 Ticket
Update platform standards to WCAG 2.2+ or WCAG 2.1 AAA | Done | Complete | Beth, DK, Eli (no longer with VA) | July 2024 Ticket
Publish a11y guide for product teams to Platform | Done | Complete | Jamie, Sara, Martha | July 2024 Ticket
Publish guide for new a11y specialists to GitHub | Done | Complete | Jamie, Sara | July 2024 Ticket
Create a slackbot based on github labels to announce design intent and midpoint reviews.a | Done | Complete | Beth | July 2024 Ticket
Publish screen reader + research troubleshooting instructions | Done | Complete | Jamie, Sara & SK | January 2026 Ticket
Publish how we do what we do so others can learn from us | Done | Complete | Accessibility Community of Practice | January 2026 Ticket

### Guidance & Documentation
_As of January 2026_

Item | 2026: Feasible? | 2026: Priority | 2026: Status | Ticket | Ticket/Link
-- | -- | -- | -- | -- | --
Publish screen reader + research troubleshooting instructions | Yes | Medium | Completed | Ticket closed | 98640
Create guidance, methods, and tooling for accessible wireframes and UAT |   |   |   |   |  
Operationalize a list of generative questions for research with disabled Veterans |   |   |   |   |  
Publish how we do what we do so others can learn from us | Yes | Medium | Completed | Ticket closed | 112113
Update a11y specialist ticketing templates and process documentation. | Yes | High | In progress | Ticket created | 131157 and 130767

### Programs & Framework
_As of January 2026_

Item | 2026: Feasible? | 2026: Priority | 2026: Status | Ticket | Ticket/Link
-- | -- | -- | -- | -- | --
Create a VA.gov a11y maturity model for teams |   | Medium | Icebox | Ticket created | 56240
Create a participatory co-design program for deeper work with disabled Veterans and project teams |   | Discovery work needed to determine priority |   |   |  
A11y month |   | Discovery work needed to determine priority |   |   |  

### Research & Discovery
_As of January 2026_

Item | 2026: Feasible? | 2026: Priority | 2026: Status | Ticket | Ticket/Link
-- | -- | -- | -- | -- | --
Conduct research and improve where we store resources, documentation, and more. | Yes | Medium | In progress | Ticket created | 44845 plus past ADE tickets
Investigate how disabled Veterans report accessibility issues |   | Discovery work needed to determine priority |   |   |  
Analyze 555 open a11y tickets (DOMO) |   | Discovery work needed to determine priority |   |   |  
Conduct generative, service-level qualitative research on the blinded Veteran experience |   |   |   |   |  

### Tools & Infrastructure
_As of January 2026_

Item | 2026: Feasible? | 2026: Priority | 2026: Status | Ticket | Ticket/Link
-- | -- | -- | -- | -- | --
Audit and create a list of inaccessible internal tools, websites, and resources |   |   |   |   |  
Add an "accessibility" option to the platform support bot |   | Discovery work needed to determine priority |   |   |  
VA.gov performance dashboard |   | Discovery work needed to determine priority |   |   |  


### Training & Education
_As of January 2026_

Item | 2026: Feasible? | 2026: Priority | 2026: Status | Ticket | Ticket/Link
-- | -- | -- | -- | -- | --
Accessibility focused training for content writers using Drupal |   | Discovery work needed to determine priority |  

cc: @artsymartha68
